### DISCUSSION TITLE
v3.3.0

### DISCUSSION THREAD

**Discussion by charlesabarnes** [bullhorn/career-portal#395] [General]
- fix a spacing issue in the location chips
- add additional spacing to fix apply issues on mobile (#390) & (#361)

<hr /><em>This discussion was created from the release <a href='https://github.com/bullhorn/career-portal/releases/tag/v3.3.0'>v3.3.0</a>.</em>

**Comment by Eryter11b** [comment]
Having a issue with uploading this to Wordpress using method 2. It faults every time. I'm using method 2 because I need to add a salary field in USD format which (from my understanding) needs to be done using method 2. Any ideas on how to fix this issue?

**Reply by charlesabarnes** [reply]
can you provide more details on the ask?

**Reply by Eryter11b** [reply]
So I'm working on a clients Wordpress site that uses bullhorn. Wordpress has the bullhorn plugin but it is limited in changes. Therefore I have to manually change the code to where users can see salary for jobs. That's the easy part.... But I cannot manually upload the Bullhorn plugin to Wordpress.

**Reply by charlesabarnes** [reply]
oh cool, have you built the code yet?  Once you built it, you should be able to replace the files in the plugin directory via your file transfer software of choice.

**Comment by Eryter11b** [comment]
I have not. I'll work on that now. Thank you for the help!
